test/colony-network-mining.js

```diff
@@ -2444,15 +2444,29 @@ contract("ColonyNetworkMining", accounts => {
         // These have to be done sequentially because this function uses the total number of tasks as a proxy for getting the
         // right taskId, so if they're all created at once it messes up.
       }
+      await fundColonyWithTokens(metaColony, clny, INITIAL_FUNDING.muln(30));
 
-      // We need to complete the current reputation cycle so that all the required log entries are present
       await advanceMiningCycleNoContest({ colonyNetwork, test: this });
 
+      for (let i = 0; i < 10; i += 1) {
+        await setupFinalizedTask( // eslint-disable-line
+          {
+            colonyNetwork,
+            colony: metaColony,
+            token: clny,
+            workerRating: 2,
+            managerPayout: 10000000,
+            evaluatorPayout: 10000000,
+            workerPayout: 10000000
+          }
+        );
+      }
+
       const clients = await Promise.all(
         accounts.slice(3, 11).map(async (addr, index) => {
           const client = new MaliciousReputationMinerExtraRep(
             { loader: contractLoader, minerAddress: addr, realProviderPort: REAL_PROVIDER_PORT, useJsTree },
-            accounts.length - index,
+            accounts.length - index - 5,
```

Comment:
For me this is the actual line that fixes the test. We should extract that variable to be clear but it is basically the `entryToFalsify` in the bad client.